scripts/scripts.js

```diff
@@ -42,13 +42,34 @@ async function loadFonts() {
   }
 }
 
+function decorateVideoLinks(main) {
+  [...main.querySelectorAll('a')]
+    .filter(({ href }) => !!href)
+    .forEach((link) => {
+      if (link.href.includes('youtube.com/embed/') || link.href.includes('youtu.be/')) {
+        const videoId = new URL(link.href).pathname.split('/').pop();
+        const iframe = document.createElement('iframe');
+        iframe.classList.add('youtube-video');
+        iframe.width = 560;
+        iframe.height = 315;
+        iframe.src = `https://www.youtube-nocookie.com/embed/${videoId}?rel=0`;
```

Comment:
good to know `youtube-nocookie`

Reply:
youtube created this code on the share screen.